src/lucky/assignable.cr

```diff
@@ -98,7 +100,7 @@ module Lucky::Assignable
           {% var = declaration.var %}
           {% type = declaration.type %}
           {% value = declaration.value %}
-          {% value = nil if type.stringify.ends_with?("Nil") && !value %}
+          {% value = nil if type.stringify.ends_with?("Nil") && value.nil? %}
```

Comment:
I changed this because if for some reason you did `param thingable : Bool? = false` it would default the value to `nil`. Probably a non-issue, but why not? :man_shrugging: